autoupdate_miner_steps.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Thank you to Namoray of SN19 for their autoupdate implementation!
+# THIS FILE CONTAINS THE STEPS NEEDED TO AUTOMATICALLY UPDATE THE REPO
+# THIS FILE ITSELF MAY CHANGE FROM UPDATE TO UPDATE, SO WE CAN DYNAMICALLY FIX ANY ISSUES
+
+conda activate bitmind
+pip install -e .
+echo "Autoupdate steps complete :)"
```

Comment:
[Question] How is this auto-updating? same with the validator steps, it seems like just updating dependencies

Reply:
in the python script, it checks whether new stuff is in the repo, and if it does it pulls. Only thing left to do is install python dependencies. 

Sn19 uses this approach so they can easily add steps ad hoc from update to update (which we may want to do for validators if we want to pre-download a new model)